### PR TITLE
Feature/redux example

### PR DESCRIPTION
**Pull request by adamcronin42**
Adding a redux example to our browser example app, a simple way to run the app (via serve), and some styling updates. To access the new redux example, click the 'Redux demo' button and you will see this screen:
image

### CODE REVIEW

File: examples/sdk/browser/src/index.ts [hunk 1]
```diff
@@ -14,6 +15,40 @@ const client = BacktraceClient.initialize({
...
+    reducers: {
+        setCount(state, action) {
+            state.count = action.payload;
+        },
+    },
+});
+
+const { setCount } = counterSlice.actions;
+
+const store = configureStore({
```
Comment (konraddysput) [inline examples/sdk/browser/src/index.ts:38]: isn't this a react specific library? Do we need to add it to the package.json?

**Review by konraddysput — APPROVED** [review]

File: examples/sdk/browser/src/index.ts (hunk 1 above)
Comment (adamcronin42) [reply examples/sdk/browser/src/index.ts:38]: I'm not quite sure what you mean. Redux.js is UI/framework agnostic (but it does tend to be used the most with react). All of this redux functionality is included in `@reduxjs/toolkit`. Is that what you mean?

Comment (konraddysput) [reply examples/sdk/browser/src/index.ts:38]: This is what I mean - we do not have @reduxjs/toolkit inside the package.json definition

Comment (adamcronin42) [reply examples/sdk/browser/src/index.ts:38]: It's there. It's a dependency in the browser example package.json.

**Review by konraddysput — APPROVED** [review]

**Pull request merged**